1-Introduction/1-intro-to-ML/translations/README.es.md

```diff
@@ -0,0 +1,113 @@
+# Introducción al machine learning
+
+[![ML, IA, deep learning - ¿Cuál es la diferencia?](https://img.youtube.com/vi/lTd9RSxS9ZE/0.jpg)](https://youtu.be/lTd9RSxS9ZE "ML, IA, deep learning - ¿Cuál es la diferencia?")
+
+> 🎥 Haz clic en la imagen de arriba para ver un video donde se discuten las diferencias entre el machine learning, la inteligencia artificial, y el deep learning.
+
+## [Cuestionario previo a la conferencia](https://white-water-09ec41f0f.azurestaticapps.net/quiz/1/)
+
+### Introducción
+
+Bienvenid@ a este curso acerca del machine learning (ML) clásico para principiantes! Ya sea que tú seas completamente nuev@ en este tema, o un practicante experimentado del ML que busca refrescar los conocimientos en un área, ¡estamos contentos de que te nos unas! Queremos crear un punto de lanzamiento amigable para tus estudios de ML y nos encantaría evaluar, responder, e incorporar tu [retroalimentación](https://github.com/microsoft/ML-For-Beginners/discussions).}
```

Comment:
I'm not sure we are using the convention of using `@` to include two genderized words in one?

Reply:
The `@` symbol is contested because among other things it adds to the gender binary, so excludes non binary people.